tests/sqlparser_mysql.rs

```diff
@@ -4247,3 +4247,33 @@ fn test_create_index_options() {
         "CREATE INDEX idx_name ON t(c1, c2) USING BTREE LOCK = EXCLUSIVE ALGORITHM = DEFAULT",
     );
 }
+
+#[test]
+fn parse_invisible_column() {
```

Comment:
Since it has now become dialect agnostic I think we can move the test to common.rs and use `verified_stmt()` there so that it runs on all dialects

Reply:
Done.